assemblies/prd-ce/src/main/resources/resource/resources/log4j2.xml

```diff
@@ -1,12 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Configuration status="WARN">
   <Appenders>
-    <RollingFile name="FILE" fileName="logs/prd.log" filePattern="logs/prd.log.%i" append="true">
+    <RollingFile name="FILE" fileName="${user.home}/.pentaho/logs/prd.log" filePattern="${user.home}/.pentaho/logs/prd.log.%i" append="false">
```

Comment:
The user.home variable has to include a sys prefix as shown here:  https://github.com/pentaho/mondrian/blob/master/assemblies/psw-ce/src/main/resources/log4j2.xml#L22